scripts/lib/CIME/test_scheduler.py

```diff
@@ -526,8 +528,24 @@ def _xml_phase(self, test):
                     envtest.set_test_parameter("STOP_OPTION",stop_option[opt])
                     opti = match.group(2)
                     envtest.set_test_parameter("STOP_N", opti)
+
                     logger.debug (" STOP_OPTION set to {}".format(stop_option[opt]))
                     logger.debug (" STOP_N      set to {}".format(opti))
+
+                elif opt.startswith('R'):
```

Comment:
@jedwards4b and think you said we don't need this "R" option but its doing more then setting mpi-serial.

Reply:
I think that the R is fine, but the hardcoded A97 case at lines 545-547 are not.   On the cesm side these have been moved to components/cam/cime_config

Reply:
Which file in cam/cime_config does this go?

Reply:
config_compsets.xml